cellprofiler_core/utilities/image.py

```diff
@@ -207,25 +208,31 @@ def make_dictionary_key(key):
 
 def image_resource(filename):
     try:
-        if os.path.join("CellProfiler","docs") in os.path.abspath(os.curdir):
-            #We're probably trying to buld the manual
-            abspath = os.path.relpath(
+        if os.path.join("CellProfiler", "docs") in os.path.abspath(os.curdir):
+            # We're probably trying to buld the manual
+            thepath = os.path.relpath(
                 os.path.abspath(
```

Comment:
“thepath” is a bad variable name lol

Reply:
Sometimes it's an abspath, sometimes it's a relpath... So it's thepath